src/wifi.rs

```diff
@@ -1897,6 +1973,8 @@ where
         self.wifi.get_scan_result()
     }
 
+    /// Waits for [`WifiEvent`]s. The wait ends when a wifi event happens, and
```

Comment:
Ditto here: you might want to reword similarly to the suggestion in `BlockingWifi`.